src/Formless/Action.purs

```diff
@@ -274,6 +275,19 @@ submit :: forall v. Variant (submit :: Unit | v)
 submit =
   inj (Proxy :: _ "submit") unit
 
+-- | Submit the form, which will trigger a `Submitted` result if the form
+-- | validates successfully. Calls `preventDefault` (`Web.Event.Event`) on the
+-- | event
+-- |
+-- | ```purescript
+-- | [ HE.onSubmit F.submitPreventDefault ]
+-- | ```
```

Comment:
```suggestion
-- | Submit the form, calling `preventDefault` from `Web.Event.Event` on the 
-- | submission event to prevent the browser from refreshing the page.
-- |
-- | ```purescript
-- | HH.form
-- |   [ HE.onSubmit F.submitPreventDefault ]
-- |   [ ... ]
-- | ```
```